wordpress-mobile/WordPress-iOS
# History / Revisions: add loading empty states

Refs. #10304 

This PR adds 2 empty state to the revisions list. When a user opens the revisions list for the first time, an empty state will appear while the manager is loading. If there are no revisions or if there's a problem, then the no result empty state appears. The reload button allows the user to try to load the revisions again

![revisions-list-empty-states](https://user-images.githubusercontent.com/912252/48628969-58e94a80-e9b0-11e8-8e58-8e1caf46e3e6.jpg)

**To test:**
- Open a revisions list for the first time. You should see the empty state.
- If an error occurs then you should see the no result empty state with the try again button

@iamthomasbishop can you let me know what do you think about the text displayed?
